app/src/main/res/layout/search_favorites_widget_search_bar_dark.xml

```diff
@@ -41,5 +42,6 @@
     <ImageView
         android:id="@+id/search"
         style="@style/SearchWidgetSearchIcon"
-        android:src="@drawable/ic_loupe_24dp_dark" />
+        android:src="@drawable/ic_find_search_24"
+        app:tint="@color/white84" />
```

Comment:
```suggestion
        app:tint="?attr/daxColorWhite" />
```

Reply:
`daxColorWhite` is white instead white84